keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -1225,6 +1226,97 @@ Note that given the estimation for duration of processing the requests is
 automatically corrected (both up and down), there is no need to change that
 in the initial version.
 
+
+#### Estimate width of LIST request
```

Comment:
BTW - let's not introduce another section for it and incorporate it in the above one (determining the width)